tests/installation/disable_grub_timeout.pm

```diff
@@ -62,7 +64,7 @@ sub run {
     send_key 'alt-t';
     my $timeout = "-1";
     # SLE-12 GA only accepts positive integers in range [0,300]
-    $timeout = "60" if !(sle_version_at_least('12-SP1'));
+    $timeout = "60" if is_sle && !(sle_version_at_least('12-SP1'));
```

Comment:
remove the `()` around the sle_version_at_least -> `if is_sle && sle_version_at_least('12-SP1')`